### DISCUSSION TITLE
HD-Hud "Restoration Project"

### DISCUSSION THREAD

**Discussion by 01kAzGuRu52** [leoetlino/project-restoration#246] [Q&A]
Hey,

when I try to use the latest version of https://restoration.zora.re/hd-hud (v110 , for PAL ROM) in Citra, together with the latest version of the HD-Texture Pack (https://de.henrikomagnifico.com/zelda-majoras-mask-3d-4k) , the rom fails to load in citra. 

Can you provide the textures for the HD-Hud seperately, so I can integrate them in my working built?

Thank you

**Comment by leoetlino** [comment]
The HD HUD mod is made up of three components:

1. ExeFS (ips/bps + exheader): a special build of Project Restoration with additional code for the new HUD
2. RomFS: layout archives (for the UI modifications)
3. optional HD textures to make the HUD look better

The main difference between a regular build of Project Restoration (the one that's linked from the main page) and the HD HUD build is that the code (1) is different, and that extra code also requires modified layout files (2) to work. The textures (3) are recommended but not required, and in fact Project Restoration can't even tell if you're using the custom textures!

I haven't used the texture pack you mention, but you should be able to make it work with the HD HUD mod, provided that you are using (1) and (2) from the HD HUD mod.
